drivers/ethernet/eth_mcux.c

```diff
@@ -51,7 +51,7 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #define FREESCALE_OUI_B2 0x9f
```

Comment:
Looks good, just one note to commit msg. It says
`enet driver was always not providing a valid DTS node reference.`
the "always not" wording is confusing as it indicates that it succeeded sometimes, did you meant to say
`enet driver was not providing a valid DTS node reference.`

Reply:
Yes, exactly

Reply:
I've pused the fix with corrected commit message.